Training/interface/EventDrop.h

```diff
@@ -0,0 +1,81 @@
+#include "TFile.h"
```

Comment:
a lot of code duplication between `EventDrop.h` and `DataMixer.h`. I suggest adding option to the `DataMixer` whatever to mix or to drop events and unify the code base

Reply:
I have added an option so that only `DataMixer.h` is necessary, please let me know if you don't like the way I've implemented it or have any suggestions for improvement